backup.config-example

```diff
@@ -27,6 +27,10 @@ GHE_NUM_SNAPSHOTS=10
 #
 #GHE_EXTRA_SSH_OPTS=""
 
+# rsync --bwlimit value. (default no limit)
+#
+#GHE_RSYNC_BWLIMIT=1250
```

Comment:
I think this is better modelled in the same way how we have the extra SSH options. So something like `GHE_EXTRA_RSYNC_OPTS` that then gives users the option to pass in any rsync options, not just a specific flag for `bwlimit`.